### PR TITLE
Make upload and speak icon tabbable

### PR DESCRIPTION
**Pull request by compulim**
# Make upload and speak button tabbable

Make sure all buttons are tabbable, and make sure <kbd>SPACEBAR</kbd> and <kbd>ENTER</kbd> also works.

# Focus indicator for all buttons

Add focus indicator (or CSS outline) to all clickable elements.

image

### CODE REVIEW

File: src/Shell.tsx [hunk 1]
```diff
@@ -89,16 +107,27 @@ class ShellContainer extends React.Component<Props, {}> implements ShellFunction
...
-            !this.props.listening && 'inactive',
+            !this.props.listening && 'inactive'
         );
 
         return (
-            <div className={className}>
-                <input id="wc-upload-input" type="file" ref={ input => this.fileInput = input } multiple onChange={ () => this.onChangeFile() } />
-                <label className="wc-upload" htmlFor="wc-upload-input">
+            <div className={ className }>
+                <label
```
Comment (compulim) [inline src/Shell.tsx]: We have to move DOM around to show the focus indicator correctly. In short, `<label>` should wrap around `<input>` so we don't need `htmlFor` prop and ID-ing everything.

Comment (danmarshall) [reply src/Shell.tsx]: OIC. The label trick gives us a free "click" event on the file uploader.

Comment (compulim) [reply src/Shell.tsx]: Yes, until we figured out the "free click" does not works with "spacebar" or "enter". :(

File: src/Shell.tsx [hunk 2]
```diff
@@ -24,22 +24,38 @@ export interface ShellFunctions {
...
 
-class ShellContainer extends React.Component<Props, {}> implements ShellFunctions {
+class ShellContainer extends React.Component<Props> implements ShellFunctions {
     private textInput: HTMLInputElement;
     private fileInput: HTMLInputElement;
 
-    constructor(props: Props) {
-        super(props);
-    }
+    private _handleSendButtonKeyPress = this.handleSendButtonKeyPress.bind(this);
```
Comment (danmarshall) [inline src/Shell.tsx]: can we use lambdas for these?

File: src/scss/botchat.scss [hunk 3]
```diff
@@ -568,78 +560,105 @@
...
-        vertical-align: middle;
-    }
+        right: 0;
 
-    .wc-console label {
-        cursor: pointer;
-        display: inline-block;
-        height: 40px;
-    }
+        > * {
```
Comment (danmarshall) [inline src/scss/botchat.scss]: can we not use star?

File: src/Shell.tsx (hunk 2 above)
Comment (billba) [reply src/Shell.tsx]: Yes please.

Comment (compulim) [reply src/Shell.tsx]: Moved to Lambda.

File: src/scss/botchat.scss (hunk 3 above)
Comment (compulim) [reply src/scss/botchat.scss]: Fixed.

**Pull request merged**